### PR TITLE
updating 01_cluster_properties.mdx and 07_using_efm_utility.mdx

### PR DESCRIPTION
**Pull request by matthew123987**
## What Changed?

- In 01_cluster_properties.mdx, rephrased failover to checks when describing `primary.shutdown.as.failure
- In 07_using_efm_utility.mdx, rephased checks surrounding use of `efm promote`

### CODE REVIEW

**Review by EFM-Bobby — APPROVED** [review]
Looks ok to me. (I hope the whitespace changes are ok -- they happen when I edit and push changes also.)

File: product_docs/docs/efm/4/07_using_efm_utility.mdx [hunk 1]
```diff
@@ -108,7 +108,7 @@ Invoke the `efm encrypt` command to encrypt the database password before includi
 
 The `efm promote` command instructs Failover Manager to perform a manual failover of standby to primary.
 
-Attempt manual promotion only if the status command reports that the cluster includes a standby node that is up to date with the primary. If there is no up-to-date standby, Failover Manager prompts you before continuing.
+Attempt a manual promotion only during a maintenance window for your database cluster, and if the status command reports that all standbys in the cluster are up to date with the primary.
```
Comment (drothery-edb) [inline product_docs/docs/efm/4/07_using_efm_utility.mdx:111]:
```suggestion
Attempt a manual promotion only during a maintenance window for your database cluster and if the status command reports that all standbys in the cluster are up to date with the primary.
```

**Review by drothery-edb — APPROVED** [review]

**Comment by drothery-edb** [comment]
This looks good and is ready to be merged. Shall I do the honors?

**Comment by matthew123987** [comment]
Thanks for fixing the links. Yes please, if ok with Bobby!

**Pull request merged**